src/oidcop/session/manager.py

```diff
@@ -105,6 +101,21 @@ def __init__(
             if "ephemeral" not in sub_func:
                 self.sub_func["ephemeral"] = ephemeral_id
 
+    def load_key(self):
+        self.key = self.conf.get("password") or rndstr(24)
+        return self.key
+
+    def load_salt(self):
+        self.salt = self.conf.get("salt") or rndstr(32)
+        return self.salt
+
+    def _init_db(self):
+        Database.__init__(
+                    self,
+                    key=self.key or self.load_key(),
```

Comment:
This make it safer when one or multiple flush occours.
The schema and the init values will be always consistent, and more, the flush doesn't overwrite the class attributes values or any other thing that then MUST be rebuilted as it were before

Reply:
But the dump/(flush)/load is not there to have something being built as it where before.
It's there to assign a specific state.
If that state is some initial state that you want to rebuilt so be it, but that is just one possible usage.

Reply:
I agree to have randomness only on start and then reuse Always that One. This Is a trivial commit.

I really don't want to use a dump from the First worker but instead those parameters in the global configuration

Reply:
and more ... It's something that belongs to security. A project password/salt should not be changed by code runtime. If this change is needed it would be possibile to change this parameter in configuration (or leave it random as it would is) and do a gracefull restart of the service.

my two cents relating the feature of getting _key and _salt some private, readonly values.
We can revert, sure, but I put it there as a security feature